### PR TITLE
Remove Ready for testing component

### LINKED ISSUE #114: Remove automatic deployments
We now often have 20+ deployments on the test panel and many open pull requests in the main Specify 7 repository. 

We have found that automatic deployments are unnecessary. Testers often prefer manually selecting a database. This process does not align with our current testing process, which focuses on testing PRs based on the milestone rather than the latest PR added to the repository.

It seems logical that we should save the system resources and revert to manually deploying these instances.

### PR DESCRIPTION
**Pull request by CarolineDenis**
Fixes #114

### CODE REVIEW

**Review by maxpatiiuk — CHANGES_REQUESTED** [review]
1. this doesn't disable the automatic deployment - it only hides them on the front-end
2. you don't need to change any code to disable automatic deployments. you just have to change this config variable:

   - Set this to 0:
      https://github.com/specify/specify7-test-panel/blob/1b150b855a20ef5f9be0bbbf3494e3734fe6438f/app/const/siteConfig.ts#L14

    - Disable the webhook that notifies test panel of changes to github:
       https://github.com/specify/specify7-test-panel/?tab=readme-ov-file#configuring-webhook

       (I have just done this)

File: app/components/Dashboard.tsx [hunk 1]
```diff
@@ -81,13 +81,6 @@ export function Dashboard({
     databaseGroups,
   };
 
-  const readyForTesting = {
-    ...deploymentProps,
-    deployments: state.deployment.filter(({ branch }) =>
-      branchesWithoutPullRequests.includes(branch)
```
Comment (maxpatiiuk) [inline app/components/Dashboard.tsx:87]: do not remove this - please revert the changes in all files except for siteConfig.ts

reason: readyForTesting is not exactly the same as automatic deployments
ready for testing means deployments on branches on which there is also a PR that is ready to be tested - regardless of whether that deployment was done automatically or not

**Review by maxpatiiuk — CHANGES_REQUESTED** [review]

**Review by maxpatiiuk — APPROVED** [review]

**Comment by CarolineDenis** [comment]
@maxpatiiuk , 

the ready for testing component is still present on the test panel. Are you sure this was enough to do in order to remove it?

**Comment by maxpatiiuk** [comment]
https://github.com/specify/specify7-test-panel/issues/114 was about "Remove automatic deployments", not about "
Remove Ready for testing component"

Automatic deployments were correctly disabled by this PR - all good.
The "Ready for testing" component does not equal "Automatic deployment". "Ready for testing" just means a deployment for a branch, that has a PR that is marked as ready for testing - regardless of whether than deployment was done manually or automatically (no more automatically as of this PR)

**Pull request merged**